<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>551</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="copy_button">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>260</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>COPY!</string>
   </property>
  </widget>
  <widget class="QLabel" name="subject_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>571</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="mail_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>521</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="copy_subject">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>120</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>COPY!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="copy_mail">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>40</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>COPY!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="another_button">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>410</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>ANOTHER</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
